### DISCUSSION TITLE
"Local" Persistent Sessions for < v5.0?

### DISCUSSION THREAD

**Discussion by jcosentino11** [emqx/emqx#7696] [Q&A] [answered]
Hello! Is there any way to store persistent sessions locally (rather than external database) for versions before v5.0? 

The [docs](https://www.emqx.io/docs/en/v4.4/advanced/config-and-data-backup-restore.html#what-about-persisted-sessions) suggest that this is not possible, just wanted to make sure there wasn't some feature like [persistent_session_store](https://github.com/emqx/emqx/blob/e87fb3d9f1cf6abb3087cf9348c1d792fb15b455/apps/emqx/etc/emqx.conf#L1645) for earlier emqx versions. Thanks!

**Comment by zmstone** [comment, marked as answer]
Hi @jcosentino11
No, there is no local storage support for 4.x or earlier.
And it’s experimental in 5.0 at the current stage.
